projects/packages/sync/src/modules/class-callables.php

```diff
@@ -394,6 +394,9 @@ public function set_plugin_action_links() {
 			return;
 		}
 		$plugins = Functions::get_plugins();
+		if ( ! is_array( $plugins ) ) {
```

Comment:
Is it worth setting a transient (not updating the option) here as well, so that we don't keep following this path if the failure persists - or follow it less frequently? But with a shorter expiration, eg. 5 minutes or so?

Reply:
I was checking the timing in logs, and it looks like logs for each specific site were happening every few minutes anyway -- so setting to something like 5 minutes  doesn't seem like it would do too much. Probably not worth it then.